Content.Server/Stunnable/Components/ElectroGlovesComponent.cs

```diff
@@ -0,0 +1,20 @@
+using Content.Server.Stunnable.Systems;
+using Content.Shared.Timing;
+using Robust.Shared.Audio;
+
+namespace Content.Server.Stunnable.Components
+{
+    [RegisterComponent, Access(typeof(ElectroGlovesSystem))]
+    public sealed class ElectroGlovesComponent : Component
+    {
+        [ViewVariables(VVAccess.ReadWrite)]
+        [DataField("energyPerUse")]
+        public float EnergyPerUse { get; set; } = 350;
+
+        [DataField("stunSound")]
+        public SoundSpecifier StunSound { get; set; } = new SoundPathSpecifier("/Audio/Weapons/egloves.ogg");
+
+        [DataField("sparksSound")]
+        public SoundSpecifier SparksSound { get; set; } = new SoundCollectionSpecifier("sparks");
+    }
+}
```

Comment:
Copy-pasting code is generally a bad idea. This should be networked + using file-scoped namespaces but because you copied stun batons it isn't.